### ISSUE TITLE
talksapp: accept markdown-based present files for talks.godoc.org

### ISSUE THREAD

**Issue by sbinet** [golang/gddo#680]
CL https://golang.org/cl/222846/ (and following) added support for markdown-based present files (as per https://github.com/golang/go/issues/33955)

it would be great if the `present` instance running at talks.godoc.org was updated to accept them as well.

**Comment by sbinet** [comment]
@dmitshur was this the right place to file that issue?

**Comment by dmitshur** [comment]
Sorry I missed this.

This is indeed the right issue tracker for tracking the work for `talksapp`.

**Comment by sbinet** [comment]
ping?

AFAIK, it's still using the old, non-markdown compliant, code...

**Comment by dmitshur** [comment]
That is true, the work of updating `talksapp` hasn't been done. I'm not aware of someone planning to do this work soon.

Another factor to consider here is that it's not easy to deploy a new version of `talksapp` now because its deployment script hasn't been kept up to date, so resolving this issue will require also getting `talksapp` deployment fixed.

**Comment by sbinet** [comment]
thanks circling back on this.
what would involve updating that deployment script? (_ie:_ could this be done by a non-googler?)

**Comment by dmitshur** [comment]
It should be possible, although it's harder because non-Googlers won't have access to the GCP project it's currently deployed to test everything works. But it can be deployed to another project for testing purposes.

The deployment [app.yaml](https://github.com/golang/gddo/blob/051695c33a3fcf220bffc5815eebc4b0a346c983/talksapp/app.yaml) file is in talksapp directory. See commit https://github.com/golang/gddo/commit/9bd4a32950217197bb319565e843237cca7f85d2 for when it was done last time. This time, there are more changes to make because of changes between Go 1.9 and 1.11 runtimes, as documented at https://cloud.google.com/appengine/docs/standard/go111/go-differences.
